portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,34 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*; 
```

Comment:
Avoid importing entire packages -- there are some pitfalls to doing so (example: https://www.codebyamir.com/blog/pitfalls-java-import-wildcards), and also being explicit about which classes we use is better